static/css/index.css

```diff
@@ -319,4 +324,80 @@ header {
   margin-right: 10px;
 }
 
+.white {
+  background-color: #fff;
+}
+
+.footer {
+  background-image: url(../imgs/contact\ form\ shapes-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: #6070ff;
+  border-top-left-radius: 73px;
+  background-position: 30px -34px;
+  padding: 123px 0 42px 0;
+}
+
+.footer__container__parfoot {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer__container__parfoot > form {
+  width: 100%;
+}
+
+.footer__container__parfoot > h3 {
+  font-size: 32px;
+  color: #fff;
+  margin-bottom: 15px;
+}
+
+.footer__container__parfoot > p {
+  text-align: center;
+  color: #fff;
+  margin-bottom: 40px;
+}
+
+.form__control > input[type="text"],
+.form__control > input[type="email"],
+.form__control > textarea {
+  border: 0;
+  outline: 0;
+  border-radius: 5px;
+  width: 100%;
+  padding: 10px 15px;
+  margin-bottom: 15px;
+}
+
+.form__control > input[type="text"]::placeholder,
+.form__control > input[type="email"]::placeholder,
+.form__control > textarea::placeholder {
```

Comment:
**[OPTIONAL]:** To match the Figma template I suggest setting the `font-family` of the inputs to the font-family provided by the design, this way you can accomplish more unity in your design :100: :heavy_check_mark: 

**Your project:**
![image](https://user-images.githubusercontent.com/92395484/192577137-4b53a38c-860e-48ca-a534-996cdd5f1700.png)

**Figma Template:**
![image](https://user-images.githubusercontent.com/92395484/192577297-72c75b15-7f6b-48b7-92d8-2f5f0b61a981.png)